application.go

```diff
@@ -130,6 +131,17 @@ type DeleteAppOpts struct {
 	Force bool `url:"force,omitempty"`
 }
 
+// TaskStats is a container for Stats
+type TaskStats struct {
+	Stats Stats `json:"stats"`
+}
+
+// Stats is a collection of aggregate statistics about an application's tasks
+type Stats struct {
+	Counts   map[string]int     `json:"counts"`
+	LifeTime map[string]float64 `json:"lifetime"`
```

Comment:
yes, I would normally write it that way; though it seems that `lifeTime` is the json produced by the marathon API...happy to change the go field name though